File: C/1072.c
#include <stdio.h>
 
int main() {
 
    int n;
    scanf("%d", &n);
    int in = 0;
    for(int i = 1; i <= n; i++){
        int num;
        scanf("%d", &num);
        if(num >= 10 && num <= 20){
            in = in + 1;
        }
    }
    
    printf("%d in\n", in);
    printf("%d out\n", n-in);
}

File: Python/1072.py
n = int(input())
nums = []
in_n = 0
out = 0

for i in range(n):
    x = int(input())
    nums.append(x)

for num in nums:
    if num >= 10  and num <= 20:
        in_n += 1
    else:
        out += 1

print(f'{in_n} in')
print(f'{out} out')